packages/blocks/src/api/parser/index.js

```diff
@@ -228,6 +230,7 @@ export function parseRawBlock( rawBlock ) {
 		parsedInnerBlocks
 	);
 	parsedBlock.originalContent = normalizedBlock.innerHTML;
+	parsedBlock.sourceMarkup = normalizedBlock.sourceMarkup;
```

Comment:
It's not clear to me what's the difference between sourceMarkup and originalContent. As far as I remember, originalContent was introduced for the same reasons.

Reply:
The introduction of `originalContent` in #14443 was ironically made to support inner blocks even though we've got a few years to reflect on it and [confirm](https://github.com/WordPress/gutenberg/pull/14443#issuecomment-478716522) it breaks them. 🙃

I find `originalContent` as a name confusing particularly because it stands in sharp contrast to how we define "content" in `innerContent`. That is, `innerContent` is an array that specifically represents the view of the document including the inner blocks while `originalContent` is a string that specifically represents the view where the inner blocks are missing.

I considered removing `originalContent` altogether but I'm worried about what might be using it. As noted in the original PR we should be moving towards removing all use of `innerHTML` everywhere as it's an historical artifact going back to an original design question revolving around two to represent nesting.

`sourceMarkup` though has a clear referent that's not muddied by the confusing terminology (confusing not only because it means the opposite of what you should infer based on the block API but also because it implies it came from the original document when in fact it was re-generated on the fly from possibly faulty inputs).

With this change in we should be able to start removing references to `originalContent` and if possible, I'd rather do that in other work since this PR is fixing a longstanding issue and I'm nervous that if we start updating and changing other parts we up the risk of introducing regressions.

Reply:
Ok, so the intent is indeed to change the way we compute `originalContent` and the name change is there to bring clarity. 

I think bringing clarity is a great argument but I wonder if we should just continue using `originalContent` that way we're not stuck supporting this old property forever and add some useless memory consumption to our stores. I'm hesitant personally here.

Reply:
If we really want to stick with the renaming, we could potentially make `originalContent` a getter/setter property in order to add a `deprecated` call maybe.

Reply:
if you're uncomfortable keeping both I'd be in favor of removing `originalContent` entirely even at risk of breaking things. I'll check the usage in the missing block and in other places. `innerHTML` should go away too but I'm not going to change that here.